src/tests/output-json-stats.c

```diff
@@ -23,7 +23,7 @@
 
 static int OutputJsonStatsTest01(void)
 {
-    StatsRecord global_records[] = { { 0 }, { 0 } };
```

Comment:
any reason for this change?

Reply:
I'm guessing for convention with the terms/names used in `OutputJsonStatsTest02`